controllers/nginx/Makefile

```diff
@@ -54,7 +54,9 @@ endif
 #        QEMUARCH=s390x
 #endif
 
-TEMP_DIR := $(shell mktemp -d)
+ifndef TEMP_DIR
```

Comment:
we already tried this in a different PR and does not work. 
Please make sure you run all the task in the makefile. This fails building the docker images

Reply:
@aledbf oh ok, let's just leave these doc issues for now. I'm going to close this PR.